ncm-metaconfig/src/main/metaconfig/logstash/pan/config_1.2.pan

```diff
@@ -0,0 +1,13 @@
+unique template metaconfig/logstash/config_1.2;
+
+variable METACONFIG_LOGSTASH_VERSION ?= '1.2';
+include format('metaconfig/logstash/schema_%s', METACONFIG_LOGSTASH_VERSION);
```

Comment:
it makes no sense that you can change the schema version in a config called `config_1.2`.
make one generic `config.pan` with this structure and default variable set to `5.0`. same for forwarder.